packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -111,19 +119,42 @@ const inputButton = style<PickerButtonProps | AriaSelectRenderProps>({
   alignItems: 'center',
   height: 'control',
   transition: 'default',
-  columnGap: 'text-to-control',
-  paddingX: 'edge-to-text',
+  columnGap: {
+    default: 'text-to-control',
+    isQuiet: 'text-to-visual'
+  },
+  paddingX: {
+    default: 'edge-to-text',
+    isQuiet: centerPadding()
+  },
   backgroundColor: {
     default: baseColor('gray-100'),
     isOpen: 'gray-200',
-    isDisabled: 'disabled'
+    isDisabled: 'disabled',
+    isQuiet: 'transparent'
   },
   color: {
     default: 'neutral',
     isDisabled: 'disabled'
+  },
+  maxWidth: {
+    isQuiet: 'max'
   }
 });
 
+const quietFocusLine = style({
+  width: 'full',
+  // Use pixels since we are emulating a border.
+  height: size(2),
```

Comment:
using size will adjust to font size and will get bigger for touch, if this was copied from somewhere else, it probably needs to be adjusted there as well
```suggestion
  height: `[2px]`,
```

Reply:
ohhh lol it was [2px] before and then i changed it size(2) but i see okay i'll change it back

Reply:
oh thats my bad, I thought the focus rings had adaptive sizing via the outline values but I see now that they just convert the `2` into the token value which is static